.github/workflows/nemo_tests.yml

```diff
@@ -151,6 +151,35 @@ jobs:
         gist_id: a4049a0fc0a0a11651a5ce6a04d76160
         file_path: ../../../../store_results/performance_history_openacc_kernels_gpu
 
+    # PSyclone, compile and run MetOffice NEMO with OpenACC kernels for GPUs
+    - name: NEMO MetOffice OpenACC loops for GPU
+      run: |
+        source .runner_venv/bin/activate
+        export PSYCLONE_NEMO_DIR=${GITHUB_WORKSPACE}/examples/nemo/scripts
+        export NEMO_DIR=${HOME}/NEMO
+        cd examples/nemo/scripts
+        make -j 4 openacc_loops
+        module load nvidia-hpcsdk netcdf_fortran
+        COMPILER_ARCH=linux_nvidia_acc_gpu make -j 4 compile-openacc_loops
+        export NV_ACC_POOL_THRESHOLD=75
+        make run-openacc_loops |& tee output.txt
+        # Check the output is as expected for the first 6 digits
+        tail -n 1 output.txt | grep -q " it :      10"
+        tail -n 1 output.txt | grep -q "|ssh|_max:  0.259483"
+        tail -n 1 output.txt | grep -q "|U|_max:  0.458515"
+        tail -n 1 output.txt | grep -q "S_min:  0.482686"
+        tail -n 1 output.txt | grep -q "S_max:  0.407622"
+        export VAR_TIME=$(grep -A 1 "Elapsed Time" output.txt | head -n 2 | tail -n 1)
+        echo $GITHUB_REF_NAME $GITHUB_SHA $VAR_TIME >> ${HOME}/store_results/performance_history_openacc_loops_gpu
+        # Upload the results into a Github Gist
+
+    - name: Upload NEMO OpenACC loops GPU results
+      uses: exuanbo/actions-deploy-gist@v1
+      with:
+        token: ${{ secrets.GIST_TOKEN }}
+        gist_id: a4049a0fc0a0a11651a5ce6a04d76160
+        file_path: ../../../../store_results/performance_history_openacc_loops_gpu
```

Comment:
This is right, the loops version is almost always faster (unless it can not parallelise some region, which we could then fallback to the kernels but I have not done this here yet). It is still 2s slower than OpenMP but it is all due to the 4 additional files that the script is filtering.